src/riak_kv_wm_aaefold.erl

```diff
@@ -560,6 +564,48 @@ malformed_range_repl_request(Filter0, QN0, RD, Ctx) ->
             {false, RD, Ctx#ctx{query= Query}}
     end.
 
+%% @private validate and parse a range repair request
+%% Needs a bucket as minimum - plus also maybe a key range and a
+%% date range
+-spec malformed_range_repair_request(#wm_reqdata{},
+                                    context()) ->
+                                        {boolean(), #wm_reqdata{}, context()}.
+malformed_range_repair_request(RD, Ctx) ->
+    case wrq:path_info(bucket, RD) of
```

Comment:
if you change to the more conventional `buckets` then change this key search

Reply:
This is set in riak_kv_web:raw_dispatch/1 - the path is buckets, but the atom for the extract from the path is `bucket`